cloudify_agent/installer/operations.py

```diff
@@ -86,10 +100,12 @@ def start(cloudify_agent, **_):
     agent_alive = utils.is_agent_alive(cloudify_agent['queue'], client)
 
     if not agent_alive:
+        utils.update_agent_record(agent_name, AgentState.NONRESPONSIVE)
```

Comment:
waiting to start is a normal stage during the installation, i'm not sure if we need to set nonresponsive right off the bat

Reply:
I agree that it can happen in the normal stage and then it's misleading.
So I want to change it, but not sure how would I know if it is really nonresponsive?